src/server/services/helpers/getDuplicates.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function getDuplicates(cleanedArray) {
+    let duplicatesObject = {};
+    duplicatesObject[cleanedArray[0][0]] = cleanedArray[0][1];
```

Comment:
are you sure that `cleanedArray` is two-dimensional? if `cleanedArray[0]` is undefined than `cleanedArray[0][0]` from undefined throw an error

Reply:
add some check before to be sure that cleanedArray is two-dimensional

Reply:
It is an array of arrays. cleanedArray[0] = ["Key": value] BUT cleanedArray[0][0] = ["Key"]